pkg/util/errutil/errhttp/writer.go

```diff
@@ -43,10 +54,48 @@ func Write(ctx context.Context, err error, w http.ResponseWriter, opts ...func(E
 
 	logError(ctx, gErr, opt)
 
+	var rsp any
```

Comment:
I would rather suggesting using something similar to [respons.ErrOrFallback](https://github.com/grafana/grafana/blob/b5d74ac200579876a7eb391fc1d8593980148cdd/pkg/api/response/response.go#L288), but that package has some "weird" dependencies on web package. So lets skip for now since this is fine, but later I would like us to revisit this.